pom.xml

```diff
@@ -11,7 +11,7 @@
 
   <groupId>org.jvnet.hudson.plugins</groupId>
   <artifactId>thinBackup</artifactId>
-  <version>1.19-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
```

Comment:
Enables [incremental builds](https://github.com/jenkinsci/incrementals-tools) in CI.